### PR TITLE
Add contact find command

### LINKED ISSUE #31: As a user, I can search the address book with keywords
... so I can easily find a specific contact

### PR DESCRIPTION
**Pull request by Nephelite**
Update UG to include contact find and contact sort 
Modify command result of contact sort.
Fixes #31.

### CODE REVIEW

File: src/main/java/taskbook/logic/commands/contacts/ContactFindCommand.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                    + ": Searches all contact names that contain the given query.\n"
+                    + "Parameters:\n"
+                    + PREFIX_QUERY + "QUERY\n"
+                    + "Only names with exact matches with QUERY will be displayed. Can be multiple words. "
+                    + "Case insensitive.\n";
+    private Predicate<Person> predicate;
+    private String query;
+
+    /**
+     * Creates a TaskFindCommand to search for tasks with the specified {@code Predicate<Task> query}.
```
Comment (dexter-sim) [inline src/main/java/taskbook/logic/commands/contacts/ContactFindCommand.java:34]: You might have forgotten to update the javadoc for this constructor.

File: src/main/java/taskbook/logic/commands/contacts/ContactFindCommand.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     * @param query
+     */
+    public ContactFindCommand(Predicate<Person> predicate, String query) {
+        requireNonNull(predicate);
```
Comment (dexter-sim) [inline src/main/java/taskbook/logic/commands/contacts/ContactFindCommand.java:38]: Would it be better if we do `requireNonNull` for query too? This way, the `equals` method would not need to check for `null`.

Comment (Nephelite) [reply src/main/java/taskbook/logic/commands/contacts/ContactFindCommand.java:38]: Good point!

File: src/main/java/taskbook/logic/commands/contacts/ContactFindCommand.java (hunk 1 above)
Comment (Nephelite) [reply src/main/java/taskbook/logic/commands/contacts/ContactFindCommand.java]: Nice catch!

**Review by dexter-sim — APPROVED** [review]
lgtm

**Pull request merged**